dist/mapbox-gl.css

```diff
@@ -5,6 +5,12 @@
     -webkit-tap-highlight-color: rgba(0,0,0,0);
 }
 
+.mapboxgl-canvas-container {
+    -webkit-transform-style: preserve-3d;
+    -moz-transform-style: preserve-3d;
+    transform-style: preserve-3d;
```

Comment:
Unfortunately [`preserve-3d` is not supported in one of our supported platforms: IE11](http://caniuse.com/#feat=transforms3d)